tests/compute/test_transform.py

```diff
@@ -95,9 +95,27 @@ def test_simple_graph():
     eset = set(zip(list(F.asnumpy(src)), list(F.asnumpy(dst))))
     assert eset == set(elist)
 
+def test_bidirected_graph():
+    def _test(in_readonly, out_readonly):
+        elist = [(0, 0), (0, 1), (0, 1), (1, 0), (1, 1), (2, 1), (2, 2), (2, 2)]
+        g = dgl.DGLGraph(elist, readonly=in_readonly)
+        elist.append((1, 2))
+        elist = set(elist)
+        big = dgl.to_bidirected(g, out_readonly)
```

Comment:
It looks like your `to_bidirected` implicitly convert a multigraph to a simple graph, which might not be the expected behavior. It's generally better to let each operator serve only one purpose, and allow the power of combination to cover rich scenarios.

Reply:
No, the returned graph is still a multi-graph.